### PR TITLE
[BE][CI] rename .jenkins to .ci, add symlink

### PR DESCRIPTION
**Pull request by janeyx99**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92621

### CODE REVIEW

**Comment by huydhn** [comment]
To enable symlink support for Windows https://github.com/pytorch/pytorch/blob/master/.github/workflows/_win-test.yml#L61-L64

**Review by huydhn — APPROVED** [review]

File: .jenkins [hunk 1]
```diff
@@ -0,0 +1 @@
+.ci
```
Comment (ZainRizvi) [inline .jenkins:1]: what does this do?

**Review by ZainRizvi — DISMISSED** [review]

File: .jenkins (hunk 1 above)
Comment (huydhn) [reply .jenkins:1]: Yeah, that how a symlink looks like on a PR, it's not a normal file though but a visual cue on where the symlink points to I think

Comment (ZainRizvi) [reply .jenkins:1]: Ah, okay. Thanks for the explanation!

**Review by ZainRizvi — APPROVED** [review]

**Comment by janeyx99** [comment]
Adding ciflow/trunk label to make sure things will pass.

Will force merge later if the only thing I'm bypassing is the pr-sanity-check.

**Comment by malfet** [comment]
@bigfootjon can you please check what is going on there?

**Comment by janeyx99** [comment]
will open another PR instead

**Pull request closed without merging**